component-maps.yml

```diff
@@ -722,3 +722,10 @@ docker_container:
     docker_image:
     - mender-gateway-qemu-commercial
     release_component: true
+
+  mender-ci-tools:
+    docker_image:
+    - mender-ci-tools
+    docker_container:
+    - mender-ci-tools
+    release_component: true
```

Comment:
Each components needs three entries: the git repo, the Docker image, and the container name. Check for example how it is done for `mtls-ambassador`

Reply:
👍